src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -890,8 +901,6 @@ private void RegisterCustomNodeDefinitionWithEngine(CustomNodeDefinition definit
                 Workspaces.OfType<HomeWorkspaceModel>().SelectMany(ws => ws.Nodes),
```

Comment:
In which case I think the name `RegisterCustomNodeDefinitionWithEngine` is not so accurate (it confused me as a reviewer), if I may suggestion an alternative to clear things up...

```csharp
/// <summary>
/// Generate graph sync data for the given custom node definition. This call 
/// does nothing if compilation of the custom node definition does not result 
/// in any dirty nodes.
/// </summary>
/// <param name="definition">
/// The custom node definition to generate graph async data for.
/// </param>
private void GenerateGraphSyncDataForCustomNode(CustomNodeDefinition definition)
{
}
```